app/v2/controllers/V2MovementsController.scala

```diff
@@ -484,6 +492,7 @@ class V2MovementsControllerImpl @Inject() (
         (for {
           _ <- ensurePositive(page.map(_.value), "page")
           _ <- ensurePositive(count.map(_.value), "count")
+          perPageCount = determineMaxPerPageCount(count.map(_.value))
```

Comment:
Should also be done for messages.

Reply:
This is done for messages at line number 438.